conf/nginx.conf

```diff
@@ -0,0 +1,30 @@
+# mysite_nginx.conf
+
+events {
+  worker_connections  1024;
+}
+
+http {
+
+  include /etc/nginx/mime.types;
+
+  upstream webapp {
+    server falaapp:8000;
```

Comment:
From what I can see, `falaapp` is defined only in `docker-compose.yml`. How does this work in Kubernetes?

Reply:
I'll finish Kubernetes work in next PR.

Reply:
If we proceed like this then it will mean we cannot deploy this application to staging any more, right? The previous image has been deleted now.

Reply:
Good point. I'll continue in this PR.

Reply:
We can also progress this pull request to the point where it's doing exactly the same as before and finish nginx off in the next one if you would like?